src/components/Thanks/SingleVersion/BadgeMilestone.vue

```diff
@@ -113,9 +106,23 @@ const showEqualityBadge = computed(() => props.isGuest || props.onlyKivaCards ||
 
 const showBadgeModule = computed(() => showEqualityBadge.value || !!props.badgeAchievedIds.length);
 
-const showSimplifiedTitle = computed(() => props.isOptedIn && !props.kivaCardsModuleShown);
+const moduleTitle = computed(() => {
+	let title = '';
+	if (props.isOptedIn) {
```

Comment:
I think we still need to check `kivaCardsModuleShown` here, since the badge module will be below the Kiva Cards module, if a transaction has both.